examples/redis/README.md

```diff
@@ -0,0 +1,106 @@
+# Overview
+
+OpenTelemetry Redis Instrumentation allows the user to automatically collect trace data and export them to the backend of choice (we can use Zipkin or Jaeger for this example), to give observability to distributed systems.
+
+This is a simple example that demonstrates tracing calls to a Redis cache via an Express API. The example
+shows key aspects of tracing such as
+- Root Span (on Client)
+- Child Span (on Client)
+- Child Span from a Remote Parent (on Server)
+- SpanContext Propagation (from Client to Server)
+- Span Events
+- Span Attributes
+
```

Comment:
Could you please explain a little more on actual example: what operations are we executing and what are the components involved?